src/main/cpp/alias/Alias.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Alias.h
+ *
+ *  Created on: Mar 11, 2017
+ *      Author: cedric
+ */
+#ifndef ALIAS_H_INCLUDED
+#define ALIAS_H_INCLUDED
+
+#include <vector>
+#include <util/Random.h>
+namespace stride {
+namespace alias {
+/**
+ * Class used to generate random numbers according to Vose's Alias Method.
+ * Used the algorithm on http://keithschwarz.com/darts-dice-coins/
+ * It uses an strides Random as random number generator.
+ */
+class Alias
+{
+    public:
+	/// No standard constructor needed
+	Alias() = delete;
+	/// No copy constructor needed
+	Alias(const Alias&) = delete;
+	/// No assignement operator needed
+	Alias& operator=(const Alias&) = delete;
+	/// Creates an Alias object. The constructor would be way too long.
```

Comment:
You might want to leave out the "constructor would be way too long." part. Comments that start with three slashes are documentation comments, and gory implementation details aren't all that relevant to the documentation.